test/e2e/app-dir/navigation/navigation.test.ts

```diff
@@ -201,11 +229,11 @@ describe('app dir - navigation', () => {
         )
       }
 
-      await checkLink(6, 94)
-      await checkLink(50, 710)
-      await checkLink(160, 2250)
-      await checkLink(300, 4210)
-      await checkLink(500, 7010) // this one is hash only (`href="#hash-500"`)
+      await checkLink(6, 108)
```

Comment:
these values updated because I added a new link to the test page, so the scroll is offset by 14px